package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof-bootstrap",
   "description": "A bootstrap for HOF projects",
-  "version": "6.0.2",
+  "version": "7.0.0",
```

Comment:
do you mean two separate PR's, one for the change log and one for the version number? Why?